jest/setupTests.js

```diff
@@ -1,3 +1,4 @@
 const { toMatchImageSnapshot } = require("jest-image-snapshot");
+const { toBeInTheDocument, toHaveClass } = require("jest-dom");
 
-expect.extend({ toMatchImageSnapshot });
+expect.extend({ toMatchImageSnapshot, toBeInTheDocument, toHaveClass });
```

Comment:
You can just use `import 'jest-dom/extend-expect';`  instead of manually extending all the things.

Reply:
Done. That also fixed a type issue